app/views/layouts/printer_friendly.html.haml

```diff
@@ -1,11 +1,13 @@
 !!!
 %html{dir:language_direction}
-  %head
+  %head      
     %title
       =h html_title
     %style
       a.anchor, a.shy {display:none;}
+      @media print { .noPrint {display: none}}
```

Comment:
I'm trying to stick to using dashes in html classes and underscores in ids. So .no-print would be prefered. Or maybe even .screen-only